### DISCUSSION TITLE
Netbox-scanner not writing the IPs and devices it found to the netbox UI

### DISCUSSION THREAD

**Discussion by jburg379** [netbox-community/netbox#17043] [Help Wanted!]
I installed netbox at the beginning of the month and added a few devices with their IPs manually the goal was always to have netbox do this automatically. I installed netbox-scanner from the github repo lopez/netbox-scanner and I am able to get it running on the terminal. I run nmap-scan.sh and it scans the network, makes the XML file and moves it to /opt/netbox/scans and then the netbox-scanner.py reads the file and it starts showing that it created the IPs and devices but nothing new shows up on netbox UI. I checked the database and found that the new IPs are not there either. I verified that the API token is correct and has the write enabled box checked. What am I missing?

**Comment by candlerb** [comment]
You're asking about a problem with some piece of third-party software (netbox-scanner.py), so best ask the author of that package.

Also read: https://netboxlabs.com/docs/netbox/en/stable/introduction/#serve-as-a-source-of-truth
